synapse/app/homeserver.py

```diff
@@ -97,9 +97,18 @@ def build_resource_for_federation(self):
         return JsonResource(self)
 
     def build_resource_for_web_client(self):
-        import syweb
-        syweb_path = os.path.dirname(syweb.__file__)
-        webclient_path = os.path.join(syweb_path, "webclient")
+        webclient_path = self.get_config().web_client_location
+        if not webclient_path:
+            try:
+                import syweb
+            except ImportError:
+                quit_with_error(
+                    "Could not find a webclient. Please either install syweb\n"
```

Comment:
Should this be "matrix-angular-sdk" rather than syweb? Should we include specific instructions on how to pip install syweb.